src/shell/lisa_gen_buildroot_rootfs

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -eu
+
+BUILDROOT_URI=git://git.busybox.net/buildroot
+BUILDROOT_VERSION=2018.08
+BUILDROOT_DIR=$LISA_HOME/src/buildroot
+BUILDROOT_PATCHES_DIR=$(dirname $0)/buildroot_patches
+BUILDROOT_CONFIG=$(dirname $0)/buildroot_config
+
+function print_usage {
+	echo "Usage: $0 [options]"
+	echo "	options:"
+	echo "		-p: purge buildroot to force a fresh build"
+	echo "		-h: print this help message"
+}
+
+while getopts "hp" opt
+do
+	case $opt in
+		h)
+			print_usage
+			exit 0
+			;;
+		p)
+			rm -rf $BUILDROOT_DIR
+			exit 0
+			;;
+		?)
```

Comment:
Is not `*)` usually used as default statement ?

Reply:
Yes usually. This is the form I found most examples using so I stuck to it too.

Reply:
If they are really the same (don't know) then perhaps just check what we most use in other LISA scripts. I guess that consistency in that case could be a way to avoid confusion (and the same question I raised before) in the future.